client/reader/stream/featured-asset.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+
+export default class FeaturedAsset extends React.Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			featuredImage: '',
+			useFeaturedEmbed: props.useFeaturedEmbed
+		};
+	}
+
+	componentDidMount() {
+		if ( this.props.featuredImage ) {
```

Comment:
likely need to eat componentWillReceiveProps and componentDidUpdate for times when the component is reused across items in the stream. Probably unlikely, but possible.